cpp_exercises/day2/date/date.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <iostream>
```

Comment:
Questa riga da sola comporta l'inclusione di 20000 righe di codice!
In questo caso possiamo cavarcela spostando l'include dentro `date.cpp` e usando una forward declaration per `ostream`.

Reply:
passato `#include <iostream>` nel _.cpp_, inserito `#include<iosfwd>` nel _.h_